### PR TITLE
Feature/create students

### PR DESCRIPTION
**Pull request by Eloi-Perez**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/user-story-template.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 I want to [action to be performed]
 So I can [result to be achieved]
 
-**__Steps to Follow__**
+****Steps to Follow****
+
 - [ ] Please add your steps here
 
-**__Additional Considerations__**
+****Additional Considerations****
```
Comment (Eloi-Perez) [inline .github/ISSUE_TEMPLATE/user-story-template.md:20]: This file was fully modified by a pre-commit during a merge

Comment (TazDeCoder) [reply .github/ISSUE_TEMPLATE/user-story-template.md:20]: It should be fine

**Review by TazDeCoder — COMMENTED** [review]
Look's good! Some things can improve on the frontend side, but this should be okay at least for now.

File: packages/backend/controllers/studentControllers.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        password: hashedPassword,
+        classroom,
+      })
+        .then(() =>
+          res.status(201).json({
+            message: 'Created Successfully',
+            fullName,
+          }),
+        )
+        .catch((err) => res.status(400).json({ message: err }));
```
[the remark this answers is not in the document]
Comment (Eloi-Perez) [reply packages/backend/controllers/studentControllers.js:33]: 400 implies user error, 500 server error

[the remark this answers is not in the document]
Comment (Eloi-Perez) [reply packages/backend/controllers/studentControllers.js:33]: I think, but I'd need to check to make sure, that on line 33 you will get an error if the mongoose validation fails (e.g. schema required: true) but probably could also be a server error if the database suddenly disconnects and is not able to save the user.

**Pull request merged**